### PR TITLE
Feat/recommendations

### PR DESCRIPTION
**Pull request by tylertruong**

### CODE REVIEW

File: database/insertionHelpers.js [hunk 1]
```diff
@@ -44,7 +43,7 @@ module.exports.addSearchQuery = (obj) => {
   const {
     searchQueryId, timestamp, market, checkIn, checkOut, roomType,
   } = obj;
-  const queryText = `INSERT INTO listings (search_id, market, searched_at, check_in, check_out, room_type) VALUES (${searchQueryId}, '${market}', '${timestamp}', '${checkIn}', '${checkOut}',  '${roomType}') ON CONFLICT (search_id) DO NOTHING`;
+  const queryText = `INSERT INTO search_queries (search_id, market, searched_at, check_in, check_out, room_type) VALUES (${searchQueryId}, '${market}', '${timestamp}', '${checkIn}', '${checkOut}',  '${roomType}') ON CONFLICT (search_id) DO NOTHING`;
```
Comment (ryanakiyama) [inline database/insertionHelpers.js]: If possible try to make queries formatted into multiple lines

File: database/insertionHelpers.js [hunk 2]
```diff
@@ -44,7 +43,7 @@ module.exports.addSearchQuery = (obj) => {
...
```
Comment (ryanakiyama) [inline database/insertionHelpers.js]: If possible, try to make long queries formatted into multiple lines

**Review by ryanakiyama — APPROVED** [review]
Long queries in one line are hard to debug and follow along, formatting them into multiple lines will make them easier to read

File: database/insertionHelpers.js (hunk 2 above)
Comment (tylertruong) [reply database/insertionHelpers.js]: Fixed it thanks!

**Review by tylertruong — COMMENTED** [review]
Thanks Ryan! Fixed it!

**Pull request merged**